index.html

```diff
@@ -0,0 +1,209 @@
+    <title>Mo Ibrahim Leadership Summit</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />    
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+</head>
+<body>
+
+<section class="pageOne">
+    <nav>
+
+        <ul class=" flex-items">
+            <li> <i class="fa-brands fa-facebook-f"></i> </li>
+            <li><i class="fa-brands fa-twitter"></i></li>
+            <li><a href="">English</a></li>
+            <li><a href="">My Page</a></li>
+            <li><a href=""></a>Logout</li>
+        </ul>
+    </nav>
+
+    <header>
+       
+        <figure class="logo">
+            <img src="images/logo.png" alt="">
+        </figure>
+        <div class="header-links">
+            <a href="about.html">About</a>
+            <a href="#program">Program</a>
+            <a href="#join">Join</a>
+            <a href="#sponsor" >Sponsor</a>
+            <a href="#news">News</a>
+            <a href="#initiatives">  </a>
+                <button id="nav-button">Our Initiatives</button> 
+        </div>
+    </header>
+
+    <div class="mobile-menu" id="show-menu">
+         <i class="fa fa-times" id="close-icon" aria-hidden="true"></i>
+       <ul id="hide-list">
+         <li>   <a href="about.html" class="toggle-menu">About</a>      </li> 
+         <li>   <a href="#program" class="toggle-menu">Program</a>  </li> 
+         <li>   <a href="#join" class="toggle-menu">Join</a>        </li> 
+         <li>   <a href="#sponsor" class="toggle-menu">Sponsor</a>  </li> 
+      </ul>
+    </div>
+    
+    
+    <div class="title">
+       
+      <div class="title-container">
+      <i class="fa-solid fa-bars" id="hamburger"></i> 
+
+          <div class="title-details">
+              <h4>"What is a government supposed to do for its people?" </h4>
+                 <div class="gradient">
+             <h1>GOVERNANCE AT THE CENTRE OF AFRICA'S DEVELOPMENT <br> 
+             <br> IBRAHIM GOVERNANCE WEEKEND <br> 2022</h1>
+             </div>
+             </div>
+
+             <div class="mobiletitle">
+             <h1> IBRAHIM GOVERNANCE WEEKEND <br> 2022</h1>
+             </div>
+
+             <div class="event-description">
+             <p>
+                The Mo Ibrahim Foundation is an African foundation, established in 2006 with one focus: the critical importance of governance and leadership for Africa.
+             </p>
+             </div>
+
+             <div class="event-time" id="time">
+             <h2>2022.5.13(FRI) ~ 15(SUN) </h2>
+             <p>Kenyatta International Convention Centre, Nairobi Kenya. </p> 
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] Kindly check on the video link below, it will show how some text disappears below the main section when on some desktop width also I noticed this effect reducing the navbar margin between the logo and the page links. [click here](https://www.loom.com/share/b67dccbddceb4cd481737e558d22ace3) to view.